hs_collection_resource/views.py

```diff
@@ -0,0 +1,106 @@
+import json
+import logging
+
+from django.http import HttpResponseRedirect, HttpResponseNotFound, HttpResponse, JsonResponse
+
+from hs_core import hydroshare
+from hs_core.views import _set_resource_sharing_status
```

Comment:
@pkdash actually i wanted to call /res_id/set-resource-flag/ api endpoint as what i did in testing code(https://github.com/hydroshare/hydroshare/pull/1000/files#diff-2bc857c60c47c3e6d0286741addd2fdeR253). 
but I dont know how to call that endpoint and how to pass parameters in POST mode in controller.
any suggestion? thanks